### PR TITLE
add a quick start guide in Example.md

### PR DESCRIPTION
**Pull request by jpurviance**

### CODE REVIEW

File: EXAMPLE.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Attending: The Docs in!
+Attending allows packages to self describe documentation through  the `__version__` and `__doc_url__` module attributes. 
+
+Attending has two example projects, `foobar` and `fizbuz`. This guid will show how to consumes docs from these example
```
Comment (ivanov) [inline EXAMPLE.md]:
```suggestion
Attending has two example projects, `foobar` and `fizbuz`. This guide will show how to consume docs from these example
```

File: EXAMPLE.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+>>> import foobar
+>>>
+>>> from attending import Library
+>>>
+>>> library = Library()
+>>>
+>>> foobar in library
+False
+>>>
+>>> library.fetch(boobar)
```
Comment (ivanov) [inline EXAMPLE.md]:
```suggestion
>>> library.fetch(foobar)
```

File: EXAMPLE.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+How do I clean up old docs?
+```python
+>>> import attending
+>>> import foobar
+>>>
+>>> from attending import Library
+>>> library = Library()
+>>> foobar in library
+True
+>>> del library[foobar] # or library.retire(foobar)
```
Comment (ivanov) [inline EXAMPLE.md:61]: I'd call it "discard" instead of "retire" in keeping with the library nomenclature.

**Review by ivanov — APPROVED** [review]
minor things, and I have more thoughts now, will submit new PRs shortly.

**Comment by ivanov** [comment]
hmm, seems I can't push to your fork, can you give me permission for that so I could fix up little things like this? Thanks

**Comment by jpurviance** [comment]
At dinner let me try from my phone.

How late do you want to work on this?

**Pull request merged**